src/element-events.js

```diff
@@ -13,6 +13,8 @@ interface EventHandler {
  * @param element
  */
 export class ElementEvents {
+  static defaultListenerOptions: boolean | AddEventListenerOptions = { capture: true };
```

Comment:
Nice. But this is unfortunately a breaking change as well, as it doesn't work for IE users. I think it should be 
`defaultListenerOptions = true`